apps/protocol-frontend/src/contexts/UserContext.tsx

```diff
@@ -173,13 +173,13 @@ export const UserContextProvider: React.FC<UserContextProps> = ({
       });
       await getUser();
       toast.success({
-        title: 'User Profile Updated',
+        // title: 'User Profile Updated',
```

Comment:
Remove rather than comment these

Reply:
Oops! Meant to remove that entirely after I added the title back in.